packages/core-sdk/src/resources/royalty.ts

```diff
@@ -287,19 +360,38 @@ export class RoyaltyClient {
       });
       await this.rpcClient.waitForTransactionReceipt({ hash });
       txHashes.push(hash);
-
-      // auto unwrap WIP tokens once they are transferred
-      if (token === WIP_TOKEN_ADDRESS && !skipUnwrapIp) {
-        const withdrawalHash = await this.wrappedIpClient.withdraw({
-          value: amount,
-        });
-        txHashes.push(withdrawalHash);
-        await this.rpcClient.waitForTransactionReceipt({ hash: withdrawalHash });
-      }
     };
     for (const { token, amount } of claimedTokens) {
       await transferToken(token, amount);
     }
     return txHashes;
   }
+
+  private async getClaimerInfo(claimer: Address) {
+    const isClaimerIp = await this.ipAssetRegistryClient.isRegistered({
+      id: claimer,
+    });
+    const ipAccount = new IpAccountImplClient(this.rpcClient, this.wallet, claimer);
+    let ownsClaimer = claimer === this.walletAddress;
+    if (isClaimerIp) {
+      const ipOwner = await ipAccount.owner();
+      ownsClaimer = ipOwner === this.walletAddress;
+    }
+    return { ownsClaimer, isClaimerIp, ipAccount };
+  }
+
+  private async unwrapWIPTokens(claimedTokens: IpRoyaltyVaultImplRevenueTokenClaimedEvent[]) {
```

Comment:
```suggestion
  private async unwrapWipTokens(claimedTokens: IpRoyaltyVaultImplRevenueTokenClaimedEvent[]) {
```
Consistent with other function names for wip.

Reply:
I observed that "wip" is consistently used throughout the code, with the exception of the WIPTokenClient class. However, the inline annotations use "WIP" instead. Do we need to ensure consistency in this regard?

Reply:
yes, WIP should be in docs. for variables, I think we can keep it as `wip` if starting name, or`Wip`.